assets/source/catalog-sync/components/OnboardingSuccessModal.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Flex,
+	Modal,
+	__experimentalText as Text, // eslint-disable-line @wordpress/no-unsafe-wp-apis --- _experimentalText unlikely to change/disappear and also used by WC Core
+} from '@wordpress/components';
+
+/**
+ * Ads Onboarding Modal.
+ *
+ * @param {Object} options
+ * @param {Function} options.onCloseModal Action to call when the modal gets closed.
+ *
+ * @return {JSX.Element} rendered component
+ */
+const OnboardingSuccessModal = ( { onCloseModal } ) => {
+	return (
+		<Modal
+			icon={
+				<img
+					src={
+						wcSettings.pinterest_for_woocommerce.pluginUrl +
+						'/assets/images/onboarding_success_modal_header.svg'
+					}
+					alt="Gift banner"
+				/>
+			}
+			onRequestClose={ onCloseModal }
+			className="pinterest-for-woocommerce-catalog-sync__onboarding-success-modal"
+		>
+			<Text variant="title.small">
+				{ __(
+					'You have successfully set up your Pinterest integration.',
+					'pinterest-for-woocommerce'
+				) }
+			</Text>
+			<Text variant="body.large">
```

Comment:
If the designs are  flexible, I'd aim for using the same body style on both modals (either `body` or `body.large`) in order to get consistency.

Reply:
I think that we should go with `body.large` as it renders as:

```css
font-weight: 400;
font-size: 16px;
line-height: 24px;
```

which is basically what I see in the Figma design.

Reply:
We will for sure do UI tweaks later next week during polishing touches.

Reply:
> Also, if you don't mind, may I push (or by yourself) a commit renaming the component AdsOnboardingModal to OnboardingModal to get some consistency too 🙂

Done.